.github/workflows/tests.yml

```diff
@@ -0,0 +1,32 @@
+name: memory_analyzer_tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Building ptr for python ${{ matrix.python-version }} on ${{matrix.os}}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+        os: [macOS-latest, ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Update pip + setuptools
+      run: |
+        python -m pip install --upgrade pip setuptools
```

Comment:
I think the most recent copy of this action is https://github.com/jreese/aioitertools/blob/master/.github/workflows/build.yml if you want something to crib.  Using `make setup` will obey whatever requirements/requrements-dev that exists in the repo.